src/components/K8s.vue

```diff
@@ -51,8 +63,56 @@ export default {
   computed: {
     cannotReset: function() {
       return (this.state !== K8s.State.STARTED && this.state !== K8s.State.READY);
+    },
+
+    memoryValueIsValid: function() {
+      let value = this.settings.kubernetes.memoryInGB;
+      let numericValue;
+      if (typeof(value) !== "number") {
+        numericValue = parseFloat(value, 10);
```

Comment:
There's no harm in running `parseFloat(<float>, 10)`, is there? Also, if I understand the UI correctly, there isn't even any way to specify a non-integral number here anyway.

Reply:
And I see `parseFloat` doesn't take a second radix argument. And this is just defensive coding right now -- I don't think there's currently a path in the code that will result in a non-numeric value, but want to guarantee that in the future.